noah/src/main/resources/kafka/kafka.conf

```diff
@@ -0,0 +1,46 @@
+#This is the configuration file for kafka
```

Comment:
Yes, all the configuration in it has a default value. I am just changing it for our purposes. The required configurations I've put as argument for the application.